lnpbp-0002.md

```diff
@@ -18,37 +18,36 @@ License: CC0-1.0
 ## Abstract
 
 The standard defines an algorithm for deterministic embedding and verification of cryptographic commitments based
-on elliptic-curve public and private key modifications (tweaks) inside the existing types of Bitcoin transaction output
-and Bitcoin scripts.
+on elliptic-curve public and private key modifications (tweaks) inside the existing types of Bitcoin transaction outputs
+and scripts.
 
 
 ## Background
 
 Cryptographic commitments embedded into bitcoin transactions is a widely-used practice. It's application include
-timestamping [1], single-use seals [2], pay-to-contract settlement schemes [3], sidechains [4], blockchain anchoring [5],
-Taproot, Graftroot proposals [6, 7, 8], Scriptless scripts [9] and many others.
+timestamping [1], single-use seals [2], pay-to-contract settlement schemes [3], sidechains [4], blockchain anchoring [5], Taproot, Graftroot proposals [6, 7, 8], Scriptless scripts [9] and many others.
 
 
 ## Motivation
 
 A number of cryptographic commitment (CC) use cases require the commitment to be present within non-P2(W)PK(H) outputs,
-which may contain multiple public keys and need a more clear definition of how the public key can be found within the
-the bitcoin script itself. For instance, embedding CC into Lightning Network (LN) payment channel state updates in the
-current version require modification of an offered HTLC and received HTLC transaction outputs [10], and these
-transactions contain only a single P2WSH output. With the following updates [11] LN will likely will change a single
-P2WPKH (named `to_remote`) output within the commitment transaction, also leading to a requirement for a standard and
-secure way of making CC inside P2(W)SH outputs.
+which may contain multiple public keys and need a clear definition of how the public key that contain the commitment
+can be found within the bitcoin script itself. For instance, embedding CC into Lightning Network (LN) payment channel 
+state updates in the current version requires the modification of an offered HTLC and received HTLC transaction outputs
+[10], and these transactions contain only a single P2WSH output. With the following updates [11] LN will likely change 
+a single P2WPKH (named `to_remote`) output within the commitment transaction, also leading to a requirement for a 
+standard and secure way of making CC inside P2(W)SH outputs.
```

Comment:
LN has already accepted scripted version of `to_remote`, so this part may be updated in the following way:

```suggestion
which may contain multiple public keys and need a clear definition of how the public key that contain the commitment
can be found within the bitcoin script itself. For instance, embedding CC into Lightning Network (LN) payment channel 
state updates in the current version requires the modification of an offered HTLC and received HTLC transaction outputs
[10], and these transactions contain only a single P2WSH output. With the use of `option_anchor_outputs` [11] all outputs in the LN commitment transaction becomes P2WSH outputs, also leading to a requirement for a 
standard and secure way of making CC inside P2(W)SH outputs.
```

Reply:
I read what I could find about `option_anchor_outputs`, iiuc if this option is activated 2 anchor outputs will be added to every commitment transaction, each with a script that makes it spendable by one participant or anyone after some time elapsed, as to make CPFP possible and ease a little fee negotiation for those commitment transactions. Hence `anchor` outputs are P2WSH, and that's the rationale why we need a scheme to find the pubkey that contains the commitment in those scripts. 

Maybe a bit nitpicking here, but I didn't know about those outputs and I just want to make sure I understand what we're talking about.